os/arch/arm/src/s5j/Make.defs

```diff
@@ -84,7 +84,7 @@ endif
 
 # Common C source files
 
-CMN_CSRCS  = up_initialize.c up_interruptcontext.c up_exit.c
+CMN_CSRCS  = up_initialize.c up_interruptcontext.c up_exit.c up_idle.c
```

Comment:
```
thapav 13 days ago  Member
s5j_pmstop() is called when generic framework decides to switch to STOP PM state (one of the states provided by SoC) whereas up_idle will be called when there is no PM framework & no PM states (architecture specific WFI mode).

Will change current implementation to:
#ifdef CONFIG_PM
pm_idle();
#else
up_idle();
#endif
```
I got your reply from my query of relationship between s5j_pmstop and up_idle.
You said, you will call up_idle when pm is not enabled. But up_idle.c is included always, even though s5j_pm is in conditional.

Reply:
Yes because after every 100ms system wakes up to execute pm_process_idle() followed by up_idle()
Now, when CPU goes from NORMAL to STOP (wfi) state, it waits for interrupt which comes with next timer tick. After this CPU executes next instruction in up_idle() & then again executes wfi. 
Similar for every tick when system wakes up to see if any other state transition is required or not, it goes back to wfi post the process to save power in between timer ticks.